lib/huginn_agent/templates/newagent/spec/spec_helper.rb.tt

```diff
@@ -0,0 +1,7 @@
+base_path = File.expand_path('../../', __FILE__)
```

Comment:
Got it, I will have a look if we can keep `STDIN` working with our `open3` approach if we can't I am oke with adding it.